packages/components/src/internal/components/search/EntityFieldFilterModal.tsx

```diff
@@ -46,8 +47,18 @@ export enum EntityFieldFilterTabs {
 const FIND_FILTER_VIEW_NAME = ''; // always use default view for selection
 
 export const EntityFieldFilterModal: FC<Props> = memo(props => {
-    const { api, entityDataType, onCancel, onFind, cards, queryName, fieldKey, showAllFields, skipDefaultViewCheck } =
-        props;
+    const {
```

Comment:
Is there any value in tracking how often user switch to "Choose Value" tab?

Reply:
Sure, added.